_posts/en/newsletters/2021-01-13-newsletter.md

```diff
@@ -0,0 +1,217 @@
+---
+title: 'Bitcoin Optech Newsletter #131'
+permalink: /en/newsletters/2021/01/13/
+name: 2021-01-13-newsletter
+slug: 2021-01-13-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes a new proposed Bitcoin P2P protocol
+message, a BIP for the bech32 modified address format, and an idea for
+preventing UTXO probing in proposed dual-funded LN channels.  Also
+included are our regular sections with the summary of a Bitcoin Core PR
+Review Club meeting, a list of releases and release candidates, and
+descriptions of notable changes to popular Bitcoin infrastructure
+software.
+
+## News
+
+- **Proposed `disabletx` message:** in 2012, [BIP37][] was published,
+  giving lightweight clients the ability to request that peers not
+  relay to the client any unconfirmed transactions until the client
+  had loaded a [bloom filter][topic transaction bloom filtering].
+  Bitcoin Core [later][bitcoin core #6993] reused this mechanism for its
+  bandwidth-saving `-blocksonly` mode where a node requests that its
+  peers don't send it any unconfirmed transactions at all.  Last year,
+  Bitcoin Core with default settings began opening a few
+  block-relay-only connections as an efficient way to improve [eclipse
+  attack][topic eclipse attacks] resistance without significantly
+  increasing bandwidth or reducing privacy (see [Newsletter #63][news63
+  bcc15759]).  However, the BIP37 mechanism being used to suppress
+  transaction relay allows the initiating node to request full
+  transaction relay at any time.  Transaction relay consumes node
+  resources such as memory and bandwidth, so nodes need to set their
+  connection limits with the assumption that each
+  BIP37-based low-bandwidth blocks-relay-only connection could suddenly
+  become a full transaction relay connection.
+
+    This week, Suhas Daftuar [posted][daftuar disabletx] to the
+    Bitcoin-Dev mailing list a new proposed BIP for a `disabletx`
+    message that could be sent during connection negotiation.  A peer
+    that understands the message won't send the node requesting
+    `disabletx` any transaction announcements and won't request any
+    transactions from the node.  It also won't send some other gossip
+    messages such as `addr` messages used for IP address announcements.
+    The `disabletx` negotiation persists for the lifetime of the
+    connection, allowing peers to use different limits for disabled
```

Comment:
This is probably my lack of understanding, but what is meant by "different limits" here? It is not clear to me.